htlcswitch/link.go

```diff
@@ -1153,6 +1159,13 @@ func (l *channelLink) processLockedInHtlcs(
 					continue
 				}
 
+				if l.cfg.DebugHTLC && l.cfg.HodlHTLC {
+					log.Errorf("hodl HTLC mode enabled, " +
```

Comment:
Warn seems like a more appropriate log level to me.

Reply:
100% agree 🎉